packages/recommend-core/src/personalization/index.ts

```
@@ -0,0 +1,42 @@
+import { getAffinities } from './getAffinities';
+import { getStrategy } from './getStrategy';
+
+type GetPersonalizationFilters = {
+  userToken: string;
+  region: string;
+  apiKey: string;
+  appId: string;
+};
+
+export const getPersonalizationFilters = async ({
+  userToken,
+  region,
+  apiKey,
+  appId,
+}: GetPersonalizationFilters) => {
+  if (!userToken || !region || !apiKey || !appId) {
+    return [];
+  }
```

Missing `region`, `apiKey` and `appId` are development errors, so they should be caught upfront and for good DX, should throw. (Ideally with `invariant()` but we don't have this dependency on this package yet.)

Missing `userToken` can be a development error or production scenario, so we could trigger a warning on top of failing silently.